src/governance/governance.cpp

```diff
@@ -1111,7 +1123,7 @@ bool CGovernanceManager::ProcessVote(CNode* pfrom, const CGovernanceVote& vote,
 
 void CGovernanceManager::CheckPostponedObjects(CConnman& connman)
 {
-    if (!::masternodeSync->IsSynced()) return;
+    if (!Assert(m_mn_sync)->IsSynced()) return;
```

Comment:
almost all usages of `m_mn_sync` is to check if blockchain is sync.

Could we instead passing all over codebase the whole object MasternodeSync just a bool flag "is blockchain synced" or "is synced"?

Reply:
or even do not call governance/masternode/etc methods externally from net_processing.cpp ?

Reply:
Could be explored in a future PR.